pkg/elemental/models/elemental.cattle.io.machineregistration.js

```diff
@@ -15,8 +15,12 @@ const DEFAULT_CREATION_YAML = `config:
       passwd: root
   elemental:
     install:
-      poweroff: true
-      device: /dev/nvme0n1`;
+      reboot: true
+      device: /dev/nvme0n1
+      snapshotter:
+        type: btrfs
+    reset:
+      enabled: true`;
```

Comment:
```suggestion
      enabled: true;
```

Reply:
@kkaempf the whole `DEFAULT_CREATION_YAML` needs to be a string, so it needs to end with `